sdk/servicebus/test-resources-post.ps1

```diff
@@ -50,44 +50,6 @@ $resourceGroup = $DeploymentOutputs['RESOURCE_GROUP']
 Write-Host "Service Bus Namespace: $namespaceName"
 Write-Host "Resource Group: $resourceGroup"
 
-# Retrieve connection strings (these contain secrets so aren't in Bicep outputs)
```

Comment:
Now that you've removed everything that uses `az` CLI, 1) you should get rid of lines 39-41 because they are unnecessary and create an additional unnecessary dependency, but then 2) why have this script at all? It does nothing useful. Just delete it.